.github/workflows/docker-build.yml

```
@@ -4,8 +4,18 @@ on:
   push:
     branches:
       - "main"
+      - "release/*"
```

```suggestion
      - "release/v0.[0-9]+.[0-9]+*"
```
@evan-forbes How does this look like?

---

yeah, makes sense